src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Security/Helpers/InsecureObjectGraphResult.cs

```diff
@@ -78,7 +77,7 @@ public Location GetLocation()
         /// Gets the display string of <see cref="InsecureSymbol"/> or <see cref="InsecureAttribute"/>.
         /// </summary>
         /// <returns>Display string of <see cref="InsecureSymbol"/> or <see cref="InsecureAttribute"/>.</returns>
-        public string GetDisplayString()
+        public string GetDisplayString(Func<TypedConstant, string> typedConstantToString)
```

Comment:
@dotpaul I went for a bit "hacky" solution but trying to have this class abstract was causing too many changes and it was hard to by-pass the static `GetOrCreate` method.